dashboard/menu/dashboard/pagecache/snapshotsettings.md

```diff
@@ -0,0 +1,19 @@
+# Settings of individual Snapshots
+
+These options can be set for each Snapshot individually by clicking on their respective rows in the table and choosing the appropriate tab of the side menu that appears.
+
+## Settings
+
+Here, you can change the name of the chosen Snapshot. You can also restrict the scope of the Snapshot using the Content Matching text boxes. These accept wildcard-compatible path specifications that set whether or not the snapshot can be used on the page(s) in question. The checkboxes allow you to set what part(s) of the site to serve from the snapshot (e.g. you can set up to cache HTML and JS/CSS but not the Images if your client wants to avoid bleedthrough on normal text but doesn’t mind it in case of images such as logos). You can also see when the Snapshot was last updated (or built) using a Scan.
+
+## Export
+
+Using this menu, you can generate a zip file containing all the pages that are in the selected snapshot. Note that this service isn’t free, it will create a cost of one proxy request per page in the page list (not per page in the snapshot). Once the export is created, you receive a link to download it via e-mail.
+
+## Copy
+
+This is where you can move the contents of your selected Snapshot into an other one. You must first select the target Snapshot from the drop-down menu then click Copy. You can then review your settings and if everything is in order click Copy again. Similarly to Export, this bears the cost of one proxy request per project page. The contents will be transferred in the background and you will receive an e-mail notification. Choosing the “Clear Target before copying” option removes all content from the Snapshot you are copying to, creating an exact copy of the original Snapshot without any pages from an older build accidentally remaining in the target.
```

Comment:
You don't _move_, you _copy_ the contents...

Reply:
Wording changed to avoid the word move in the latest commit.